packages/protocol/contracts/governance/Validators.sol

```diff
@@ -402,6 +402,8 @@ contract Validators is
     require(uptime <= FixidityLib.fixed1().unwrap(), "Uptime cannot be larger than one");
     uint256 numerator;
     uint256 denominator;
+    uint256 grace_period=6250000000000000000000;
```

Comment:
Should this be a governable state variable so we don't require source changes if we want to change it in future?

Reply:
We had a pretty long conversation on this, and I think the conclusion was that it should just pass as is and making the variable governable can be done after the fact

Reply:
@mcortesi Can you give context on whether this is a safe grace period? I.e. it's not too big to impact validator performance too much?